src/controllers/update/overseas.entity.query.controller.ts

```diff
@@ -32,7 +33,9 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
 
     const entityNumber = req.body[EntityNumberKey];
     const companyProfile = await getCompanyProfile(req, entityNumber);
-    if (!companyProfile) {
+    const privateOeDetails = await getPrivateOeDetails(req, entityNumber);
+    // TO-DO: Need to handle error for no email
```

Comment:
Should a failure to retrieve private details result in a Service Unavailable error or a Company Not Found validation error (which is already there for a failure to get companyProfile info)?

Reply:
Treat same as company profile error.